lib/versioning/poetry/index.ts

```diff
@@ -157,25 +189,38 @@ function getNewValue({
     return currentValue;
   }
   try {
-    const newSemver = npm.getNewValue({
-      currentValue: poetry2npm(currentValue),
-      rangeStrategy,
-      currentVersion: poetry2semver(currentVersion),
-      newVersion: poetry2semver(newVersion),
-    });
-    const newPoetry = npm2poetry(newSemver);
-    return newPoetry;
+    const currentSemverVersion =
+      currentVersion && poetry2semver(currentVersion);
+
+    const newSemverVersion = poetry2semver(newVersion);
+
+    if (currentSemverVersion && newSemverVersion) {
+      const newSemver = npm.getNewValue({
+        currentValue: poetry2npm(currentValue),
+        rangeStrategy,
+        currentVersion: currentSemverVersion,
+        newVersion: newSemverVersion,
+      });
+      const newPoetry = newSemver && npm2poetry(newSemver);
+      if (newPoetry) {
+        return newPoetry;
+      }
+    }
   } catch (err) /* istanbul ignore next */ {
```

Comment:
Seems like this PR fixes this try-catch thing, but let's see if `Could not generate new value using npm.getNewValue` message is still being logged or not